platform/ios/app_delegate.mm

```diff
@@ -92,8 +101,28 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 	mainViewController = viewController;
 
-	// prevent to stop music in another background app
-	[[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryAmbient error:nil];
+	int sessionCategorySetting = GLOBAL_GET("audio/general/ios/session_category");
+
+	// Initialize with default Ambient category
```

Comment:
```suggestion
	// Initialize with default Ambient category.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).